coderd/database/migrations/000223_dormancy_notification_template.up.sql

```diff
@@ -0,0 +1,16 @@
+-- TODO: autogenerate constants which reference the UUIDs
```

Comment:
Are you up-to-date with `main`? We're already up to `225`: https://github.com/coder/coder/tree/main/coderd/database/migrations